src/components/Bounties/index.tsx

```
@@ -15,30 +15,39 @@ import { chunkArray } from './utils/ChunksArr';
-	};
-
 	const handleBeforeChange2 = (next: number) => {
 		setCurrentSlide2(next);
 	};
 
-	const extendedDataChunks = extendedData ? chunkArray(extendedData.posts, isMobile ? 1 : 3) : [];
 	const activeDataChunks = activeBountyData ? chunkArray(activeBountyData.posts, isMobile ? 1 : 3) : [];
 
+	const [currentPage, setCurrentPage] = useState(1);
+	const count = extendedData?.count || 0;
+	const length = extendedData?.posts?.length;
```

ESLint

[error] 'length' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)